tests/framework/config/client.go

```diff
@@ -59,3 +59,11 @@ type LeaseOption struct {
 type UserAddOptions struct {
 	NoPassword bool
 }
+
+type WatchOptions struct {
+	Prefix      bool
+	Revision    int64
+	RangeEnd    string
+	ExecCmd     []string
```

Comment:
Please don't use ExecCmd as this breaks abstraction. Goal is to extract common tests between integration and e2e, however we should watch for tests that make sense only for one environment.